code/modules/camera/camera.dm

```diff
@@ -1,5 +1,5 @@
 /obj/machinery/camera
-	name = "security camera"
```

Comment:
This means any admin spawned security cameras will be called "autoname - SS13"
<img width="263" height="221" alt="image" src="https://github.com/user-attachments/assets/8e3e7a22-ecef-4808-8566-f355dbfba776" />
Maybe set the name according to the game state in New instead? I'm not really sure what the point of the autoname string is anyway.